pkg/configmap/configmap_writer.go

```diff
@@ -63,6 +64,32 @@ func TranslateInvalidChars(input string) string {
 	return validConfigMapKey
 }
 
+func validateAnnotationKeys(file AnnotationsFile) error {
+	errors := []error{}
+	if file.Annotations.Resources == "" {
+		errors = append(errors, fmt.Errorf("No value set for %s", bundle.ManifestsLabel))
```

Comment:
You might consider just doing `errors.New(bundle.ManifestsLabel)` and wrapping the error with an appropriate error message if this function returns with an error.

Reply:
@jpeeler is there any specific reason we'd want to use `errors.New()` instead of `fmt.Errorf()`? `fmt.Errorf()` allows us to have the words `No value set for ___` for each error, and then `strings.Join()` allows us to display all the errors in a comprehensible way: 
```
No value set for operators.operatorframework.io.bundle.manifests.v1
No value set for operators.operatorframework.io.bundle.package.v1
```
Is there a better practice for error aggregating with `errors,New()` that allows us to display the output like that?

Reply:
This is just a way to reduce code duplication and make the reported errors a little less verbose. If you return just the annotation with a problem, you can do something like:
```
if err = validateAnnotationKeys(configMapPopulate.GetAnnotations()); err != nil {
    return errorwrap.Wrap(err, "annotation validation failed, missing or empty values")
}
```
I don't know if it's really a best practice though.

Reply:
@jpeeler Updated the PR to wrap the aggregated errors. Had to vendor in the `errors` package though. PTAL